common/prettify.pm

```diff
@@ -618,47 +620,46 @@ sub Header ()
 
 sub Footer ()
 {
-    my $self = shift;
-    my $out = $self->{FH};
-
-    my $indent = '  ';
-    print $out $indent, "<testsuite name=\"Autobuild_Tests\" ";
-    if (defined $self->{TIMESTAMP} and length $self->{TIMESTAMP})
-    {
-        print $out "timestamp=\"$self->{TIMESTAMP}\" ";
-    }
-
-    my $numtests = @{$self->{TESTS}};
-
-    if ($numtests == 0)
-    {
-        # Need to insert a dummy testcase to jenkins doesn't think there is
-        # a failure.
-        print $out 'tests="1">', "\n", $indent x 2,
-          '<testcase name="dummy_test"/>', "\n";
-    }
-    else
-    {
-        print $out "tests=\"$numtests\" failures=\"$self->{FAILED}\">\n";
-    }
-
-    foreach my $test (@{$self->{TESTS}})
-    {
-        my $error = $test->{ERROR} || '';
-
-        print $out $indent x 2,
-          "<testcase name=\"$test->{NAME}\" status=\"$test->{RESULT}\" ",
-          "time=\"$test->{TIME}\"", ($error eq "" ? "/>\n" : '>');
-
-        if ($error ne "")
-        {
-            print $out "<failure>\n", $indent x 3,
-              "<![CDATA[$error]]></failure><system-out>\n", $indent x 3,
-              "<![CDATA[$test->{OUT}]]></system-out></testcase>\n";
-        }
-    }
-
-    print $out $indent, "</testsuite>\n</testsuites>\n";
+  my $self = shift;
```

Comment:
Let's be sure to preserve whatever indentation scheme  is being used by autobuild (4 spaces) rather than imposing our own scheme onto small sections of the code (2 spaces).

Reply:
Ditto this comment for other change sections in this file. It will also be more clear what actually changed vs what simple was re-indented.